### PR TITLE
Added a script to build a Heroku pipeline based on app.json

### PR DESCRIPTION
**Pull request by blarghmatey**
In order to make it easier and more repeatable to create a Heroku
environment I wrote a script that uses the Heroku toolbelt and the
`jq` command line utility to provision a pipeline with associated apps.

### CODE REVIEW

**Comment by noisecapella** [comment]
Race condition, I'll assign it back to you @giocalitri

File: {{ cookiecutter.project_name }}/bin/build_heroku_pipeline.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+APP_NAME="$1"
```
Comment (noisecapella) [inline {{ cookiecutter.project_name }}/bin/build_heroku_pipeline.sh]: Can you add a check that `$1` exists?

File: {{ cookiecutter.project_name }}/bin/build_heroku_pipeline.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (noisecapella) [inline {{ cookiecutter.project_name }}/bin/build_heroku_pipeline.sh]: We should also set the exe bit for this file

**Comment by giocalitri** [comment]
@noisecapella I'm fine with 2 reviewers

**Comment by giocalitri** [comment]
I am fine with this version, but @noisecapella has some comments too.

File: {{ cookiecutter.project_name }}/bin/build_heroku_pipeline.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+if [ -z $APP_NAME ]
+then
+    echo "Usage: bin/build_heroku_pipeline.sh some-app-name"
+    exit 1
+fi
+
+if [ -z `which heroku` ]
+then
+   echo "You need to install the Heroku CLI for this script to work. https://devcenter.heroku.com/articles/heroku-cli"
+fi
```
Comment (noisecapella) [inline {{ cookiecutter.project_name }}/bin/build_heroku_pipeline.sh]: Can you add `exit 1` here and in the `if` blocks below?

**Pull request merged**